src/main/java/com/airlines/model/Airplane.java

```diff
@@ -0,0 +1,81 @@
+package com.airlines.model;
+
+import java.time.LocalDate;
+
+public class Airplane {
+    private int id;
+    private String codeName;
+    private String model;
+    private LocalDate manufactureDate;
+    private int capacity;
+    private double flightRange;
+    private int crewId;
+
+    public Airplane() {
+    }
+
+    public Airplane(int id, String codeName, String model, LocalDate manufactureDate, double flightRange, int crewId) {
+        this.id = id;
+        this.codeName = codeName;
+        this.model = model;
```

Comment:
it could be enum, justification a little below

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/1269dee2e4effc40beaddfca6d85b7799518548f